File: week02/day_2/draw-pyramid.py
__author__ = 'ylwoi'

# Write a program that reads a number from the standard input, then draws a
# pyramid like this:
#
#
#    *
#   ***
#  *****
# *******
#
# The pyramid should have as many lines as the number was

x = int(input("Enter a number to set the piramid's size: "))
for i in range(x):
    print((x-i) * " " + (i * "*") * 2 + "*")

File: week07/day2/draw-pyramid.js
/**
 * Created by cw on 2017-05-02.
 */
'use strict';

var lineCount = 16;
var triangle = '*';
var space = ' ';

// Write a program that draws a
// pyramid like this:
//
//
//    *
//   ***
//  *****
// *******
//
// The pyramid should have as many lines as lineCount is
for (var i = 0, j = 1, k = lineCount; i < lineCount; i++, j += 2, k --) {
    console.log(space.repeat(k), triangle.repeat(j));
}